server/src/main/scala/manifests/passengers/BestManifests.scala

```diff
@@ -70,6 +70,21 @@ object BestAvailableManifest {
     )
   }
 
+  def historic(manifest: VoyageManifest): BestAvailableManifest = {
```

Comment:
Can we extract the common code from this and the `apply` above since there's only the source line that's different